pal/system-info/web/system-info.ts

```diff
@@ -168,14 +169,7 @@ class SystemInfo extends EventTarget {
                 }
             }
         }
-        let supportImageBitmap = false;
-        if (!TEST && typeof createImageBitmap !== 'undefined' && typeof Blob !== 'undefined') {
-            _tmpCanvas1.width = _tmpCanvas1.height = 2;
-            createImageBitmap(_tmpCanvas1, {}).then((imageBitmap) => {
-                supportImageBitmap = true;
-                imageBitmap?.close();
-            }).catch((err) => {});
-        }
+        const supportImageBitmap = false;  // Initialize in Promise
```

Comment:
seems there is no need to declare this cosntant variable

Reply:
can we migrate all the initialization from `constructor` to  `init`
so that all the init promise can execute at the very beginning

then here the variable `supportImageBitmap ` should be correctly initialized,
and also we can remove the  `_setFeature` method